examples/audio/eeg_signal.py

```diff
@@ -0,0 +1,527 @@
+"""
+Title: EEG Signal Classification for Stimuli identification
+Author: [Suvaditya Mukherjee](https://twitter.com/halcyonrayes)
+Date created: 2022/11/03
+Last modified: 2022/11/04
+Description: Training a Convolutional model to classify EEG signals produced by exposure to certain stimuli.
+"""
+"""
+## Introduction
+
+The following example explores how we can make a Convolution-based Neural Network to
+perform classification on EEG signals captured when subjects were exposed to different
+stimuli.
+We train a model from scratch since such signal-classification models are fairly scarce
```

Comment:
Unrelated to code review but it would be cool to offer some spectrogram specific transfer learning weights.

Reply:
I'd love to host the weights of the model I trained somewhere, but no idea how to go about that. Would be happy to work on that with some help :)
I'm assuming, by spectrogram-specific transfer learning weights, you mean just that.

Reply:
Definitely a separate project.  Do you work with spectrograms a lot?  

We should merge a ton of spectrogram datasets and have a repo where we pretrain a KerasCV backbone or two on them, then offer the weights from Huggingface or something.

Reply:
I don't, actually :sweat_smile: I'm more of an image person than spectrogram signals directly, but for this specific project as such, I'd be happy to work it out. Sounds like a great learning experience to me!